src/main/java/com/algorand/algosdk/account/Account.java

```diff
@@ -207,12 +207,6 @@ static public Transaction transactionWithSuggestedFeePerByte(Transaction copyTx,
             case KeyRegistration:
                 return new Transaction(copyTx.sender, newFee, copyTx.firstValid, copyTx.lastValid, copyTx.note, copyTx.genesisID, copyTx.genesisHash,
                         copyTx.votePK, copyTx.selectionPK, copyTx.voteFirst, copyTx.voteLast, copyTx.voteKeyDilution);
-            case AssetConfig:
-                return new Transaction(copyTx.sender, newFee, copyTx.firstValid, copyTx.lastValid, copyTx.note, copyTx.genesisID, copyTx.genesisHash,
-                        copyTx.assetID, copyTx.assetParams);
-            case AssetFreeze:
-                return new Transaction(copyTx.sender, newFee, copyTx.firstValid, copyTx.lastValid, copyTx.note, copyTx.genesisID, copyTx.genesisHash,
-                        copyTx.assetFreezeID, copyTx.freezeTarget, copyTx.freezeState);
```

Comment:
I think these need to stay, don't they? Or is that not true, since the function is deprecated?

Reply:
function is deprecated. It should not support new features.